src/zcl_aff_abap_doc_parser.clas.abap

```diff
@@ -555,6 +555,8 @@ CLASS zcl_aff_abap_doc_parser IMPLEMENTATION.
 
     DATA(string_to_parse) = abap_doc_string.
 
+    REPLACE ALL OCCURRENCES OF PCRE `\\` IN string_to_parse WITH `\\`.
+    REPLACE ALL OCCURRENCES OF PCRE `\"` IN string_to_parse WITH `\\"`.
```

Comment:
I don't think that this is the right place where the escaping should happen. The generator can also generate a simple transformation and not only JSON schema. Although we do not have pattern handling (yet?) in the transformation part, it could be possible. Therefore, I'd put the escaping into the schema generator.

Reply:
Okay, when you say schema generator, do you mean the ZCL_AFF_WRITER_JSON_SCHEMA class?

Reply:
yes, exactly